### DISCUSSION TITLE
Leaf Shearing Addon Missing

### DISCUSSION THREAD

**Discussion by Insomnik** [InnovativeOnlineIndustries/Industrial-Foregoing#876] [Feature Requests]
**Industrial Foregoing Version:**
industrial-foregoing-1.16.3-3.1.1-a834e76
**Titanium Version:**
titanium-1.16.3-3.1.0

There is no leaf-shearing addon like there was in 1.12.2

**Comment by Chreaus** [comment]
i'm missing this today too :( any word on that or some of the other handy augments? or are we supposed to shear leaves some other way now?

**Reply by quasarea** [reply]
made an issue regarding this https://github.com/InnovativeOnlineIndustries/Industrial-Foregoing/issues/979
